ansible/roles/gelato-ha-installer/scenarios/release.yml

```diff
@@ -32,3 +32,57 @@
   when:
     - gelatoexisted.stat.exists is undefined or gelatoexisted.stat.exists == false
 
+- name: Update the release version in deployment ymls
+  replace:
+    path: "{{ gelato_work_dir }}/multi-cloud/K8s/{{ item }}"
+    regexp: 'RELEASE_VERSION'
+    replace: '{{ gelato_release_version }}'
+  become: yes
+  with_items:
+    - api-deployment.yaml
+    - backend-deployment.yaml
+    - block-deployment.yaml
+    - datamover-deployment.yaml
+    - dataflow-deployment.yaml
+    - file-deployment.yaml
+    - s3api-deployment.yaml
+    - s3-deployment.yaml
+
+- name: Update the api, file, s3 deployments
```

Comment:
nit: add reference to multicloud in the job name since we also have SODA API.

Reply:
will do in followup PR.

Reply:
Will do in next PR